definition/data/sheets/CaseEventToFields.json

```diff
@@ -46,31 +46,27 @@
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "docUploadForParty", "PageID": 1, "PageDisplayOrder": 1,"DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 4, "FieldShowCondition": "directionList CONTAINS\"DOCUMENTS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "eyewitnessUploadDeadline", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 5, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "eyewitnessUploadForParty", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "OPTIONAL", "PageFieldDisplayOrder": 6, "FieldShowCondition": "directionList CONTAINS\"EYEWITNESS\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "expertReportPermissionPartyAskedByClaimant", "PageID": 1, "PageDisplayOrder": 1,"DisplayContext": "READONLY", "PageFieldDisplayOrder": 7},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "expertReportPermissionPartyAskedByDefendant", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "READONLY", "PageFieldDisplayOrder": 8},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "grantExpertReportPermission", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "MANDATORY", "PageFieldDisplayOrder": 9},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "expertReportInstruction", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "MANDATORY", "PageFieldDisplayOrder": 10, "FieldShowCondition": "grantExpertReportPermission=\"Yes\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "otherDirections", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "COMPLEX", "PageFieldDisplayOrder": 11},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "paperDetermination", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "READONLY", "PageFieldDisplayOrder": 12},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "preferredDQCourt", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "READONLY","PageFieldDisplayOrder": 13, "FieldShowCondition": "paperDetermination=\"NO\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "preferredCourtObjectingParty", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "READONLY","PageFieldDisplayOrder": 14, "FieldShowCondition": "paperDetermination=\"NO\""},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DrawJudgesOrder", "CaseFieldID": "newRequestedCourt", "PageID": 1, "PageDisplayOrder": 1, "DisplayContext": "READONLY","PageFieldDisplayOrder": 15, "FieldShowCondition": "paperDetermination=\"No\""},
```

Comment:
"paperDetermination=\"No\""} No should be changed to NO?

Reply:
line no